content/rollbear/index.md

```diff
@@ -0,0 +1,18 @@
+---
+{
+	name: "Björn Fahller",
+	firstName: "Björn",
+	lastName: "Fahller",
+	description: "I write code, I do not drink coffee.\nAvatar by Franzisca Panter",
+	socials: {
+		twitter: "bjorn_fahller",
+		github: "rollbear",
+        mastodon: "https://mastodon.social/@bjorn_fahller"
+    },
+	pronouns: "",
```

Comment:
I thought I'd seen your pronouns on one of your profiles, but wasn't sure after double-checking.

Feel free to remove this flag if you're uncomfortable disclosing

Reply:
he/him is the right pronouns.

Drop twitter. As Kevlin Henney said, "twitter is now my ex". The last few posts I made there went into the void without any response at all, so it seems like the people who follow me there aren't logging in.